python/ray/util/spark/start_ray_node.py

```diff
@@ -52,65 +58,52 @@
     # using the temp directory.
     fcntl.flock(lock_fd, fcntl.LOCK_SH)
     process = subprocess.Popen([ray_cli_cmd, "start", *arg_list], text=True)
+    ray_session_dir = os.path.realpath(os.path.join(temp_dir, "session_latest"))
+    with open(os.path.join(temp_dir, GLOBAL_RAY_CLUSTER_SESSION_NAME_FILE), "w") as f:
+        f.write(ray_session_dir)
```

Comment:
Let's simplify the code:

we can directly save the `ray_session_dir` in memory, we don't need to create a file.

Reply:
But it's possible that repl restarted

Reply:
Yes you are right.

Reply:
we can simplify it like this:
https://github.com/WeichenXu123/ray/commit/cb54ea73391aa7ba87ba21d23cc47076d6482013